xbmc/cores/VideoPlayer/Interface/StreamInfo.h

```diff
@@ -44,7 +44,8 @@ struct StreamInfo
   int bitrate = 0;
   std::string language;
   std::string name;
-  std::string codecName;
+  std::string codecName; // Codec name (name definition from ffmpeg)
+  std::string codecDesc; // Codec description
```

Comment:
i remember well the first few times i tried to understand how this code worked and it was not clear what "codec" was referred, so i thought it would be useful to have a descriptive comment on the variable since i made a small change here
i dont want touch tons of code to rename a variable that dont concern this PR,
same for the below `SelectionStream` that i add same comments,
i will remove all comments that you dont like

Reply:
removed